Assets/Scripts/Dashboard/Button.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+using Anivision.Core;
+using Anivision.PlayerInteraction;
+
+public class Button : MonoBehaviour
+{
+    public Color defaultColor;
+    public Color hoverColor;
+    [HideInInspector] public UnityEvent onClick;
+
+    private InputManager _inputManager;
+    private HapticsController _hapticsController;
+    private TeleportController _teleportController;
+
+    private void OnEnable()
+    {
+        _inputManager = InputManager.Instance;
+        if (_inputManager == null) throw new System.Exception("Must have an input manager script in the scene");
+
+        _hapticsController = HapticsController.Instance;
+        _teleportController = FindObjectOfType(typeof(TeleportController)) as TeleportController;
+    }
+
+    private void OnTriggerEnter(Collider other)
+    {
+        if (other.tag == "selector")
+        {
+            _teleportController.enabled = false;
+            _hapticsController.Haptics(1, 0.25f, 0.1f, OVRInput.Controller.RTouch);
+            ChangeColor(hoverColor);
+        }
+    }
+    private void OnTriggerStay(Collider other)
+    {
+        if (other.tag == "selector" && _inputManager.IsButtonPressed(InputManager.Button.RIGHT_TRIGGER))
```

Comment:
does this work even though you're not calling `_inputManager.IsButtonPressed` in the Update function?

Reply:
it do